### PR TITLE
Create Requirements_And_Goals.md

### PR DESCRIPTION
**Pull request by kavyashivakumar**

### CODE REVIEW

File: Requirements/Requirements_And_Goals.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+# Requirements and Goals
+
+## Mission
+
+> The Hey, Blue! initiative facilitates moments of meaningful connection between police officers and members of their community by offering intentional opportunities for individuals to meet and share positive experience.
+
+## Goal
+
+To provide a cost effective platform which helps the civilians connect with police officers for a positive interaction experience at a scale in a secure way. This platform can be hosted on both web and mobile based.
```
Comment (mgasca) [inline Requirements/Requirements_And_Goals.md:9]: `helps civilians` not `helps the civilians`
`at scale` not `at a scale`
`This platform must be accessible on both mobile and web`

File: Requirements/Requirements_And_Goals.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+## Actors
+- Citiens
+- Police officers
+- Charities
+- Retail Organisations
+
+## Functional Requirements
+
+### Hard Requirements 
+ 1. End-Use Ease of use (HR1)
```
Comment (mgasca) [inline Requirements/Requirements_And_Goals.md:20]: IMO, use of identifiers, e.g. `(HR1)` feels unnecessary

File: Requirements/Requirements_And_Goals.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+ 2. Location Tracking (HR2)
+ 3. Connecting with other users (HR3)
+ 4. Uploading to social media (HR4)
+ 5. Tracking engagements
+ 6. Ability to opt in to and out of push Notifications
+ 7. Users can only have one connection per officer per 24hour period
```
Comment (mgasca) [inline Requirements/Requirements_And_Goals.md:26]: `Civilians`not `Users`

File: Requirements/Requirements_And_Goals.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+ 8. Community members can find officers and not the other way around
```
Comment (mgasca) [inline Requirements/Requirements_And_Goals.md:27]: Be consistent, either use `civilian` or ` community member` everywhere

File: Requirements/Requirements_And_Goals.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+ 9. Officers locations shuts off after 15 minutes
+ 10. community members to be able to see how many connections each participating officer has made
+ 11. Allow Hey, Blue! to track all participation by zi cde.
```
Comment (mgasca) [inline Requirements/Requirements_And_Goals.md:30]: `zip`

File: Requirements/Requirements_And_Goals.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+ 10. community members to be able to see how many connections each participating officer has made
+ 11. Allow Hey, Blue! to track all participation by zi cde. 
+
+### Civilian Process
+
+|Id|User Story|Service|
+|--- |--- |---|
+|1|Create User Profile. (Allow deletion, updating, Hide?)|User registration|
+|2|Update, delete, Hide User Profile.|Profile Service|
+|3| Notify for an interaction with policice office within 10ft distance|Notification and Location|
```
Comment (mgasca) [inline Requirements/Requirements_And_Goals.md:38]: Police misspelled

File: Requirements/Requirements_And_Goals.md [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+|4|Establish connection with the police officer and have a virtual handshake|Handshake|
+|5|Post connections on HeyBlue! social media|right name?|
```
Comment (mgasca) [inline Requirements/Requirements_And_Goals.md:40]: Not clear what ` right name?` means here

File: Requirements/Requirements_And_Goals.md [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+|6|Update civilians account/profile with points|PointsCredition|
```
Comment (mgasca) [inline Requirements/Requirements_And_Goals.md:41]: I don't think `Credition` is a word 😄

File: Requirements/Requirements_And_Goals.md [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+|7|*allow civilians to upload the awards program of other business into Hey Blue!*|?|
```
Comment (mgasca) [inline Requirements/Requirements_And_Goals.md:42]: I don't understand this, also why is this one highlighted?

File: Requirements/Requirements_And_Goals.md [hunk 10]
```diff
@@ -0,0 +1,73 @@
...
+|8|Redeem points at participating business for goods|Redeem|
+
+### Police Officer Process
+
+|Id|User Story|Service|
+|--- |--- |---|
+|1|Create User Profile. (Only work email and load diff prefixs|User registration|
```
Comment (mgasca) [inline Requirements/Requirements_And_Goals.md:49]: What does `load did prefixes` mean?

File: Requirements/Requirements_And_Goals.md [hunk 11]
```diff
@@ -0,0 +1,73 @@
...
+|8|ability to upload upcoming events to find officers willing to connect|?|
+|9|Geo location must shut off every 15mins|Location|
+
+### Charity Process
+
+|Id|User Story|Service|
+|--- |--- |---|
+|1|Create Profile.|User registration|
+|2|Update, delete, Hide User Profile.|Profile Service|
+|3|Allow Police officers to connect|Handshake|
```
Comment (mgasca) [inline Requirements/Requirements_And_Goals.md:65]: For charity?

**Review by mgasca — COMMENTED** [review]
Added comments

File: Requirements/Requirements_And_Goals.md (hunk 2 above)
Comment (kavyashivakumar) [reply Requirements/Requirements_And_Goals.md:20]: agree. i started off and realised its of no use but forgot to delete

Comment (kavyashivakumar) [reply Requirements/Requirements_And_Goals.md:20]: also who resolves the conversation in these changes? author or collaborator?

File: Requirements/Requirements_And_Goals.md (hunk 4 above)
Comment (kavyashivakumar) [reply Requirements/Requirements_And_Goals.md:27]: Thats the language coming from the requirement.

File: Requirements/Requirements_And_Goals.md (hunk 7 above)
Comment (kavyashivakumar) [reply Requirements/Requirements_And_Goals.md:40]: That was something I am trying to name under what module/component/service comes in. what do we call a service which takes care of publishing to sociall media

File: Requirements/Requirements_And_Goals.md (hunk 8 above)
Comment (kavyashivakumar) [reply Requirements/Requirements_And_Goals.md:41]: Think these names need to me in accordance  with what service names we decide. Any better name for now?

File: Requirements/Requirements_And_Goals.md (hunk 9 above)
Comment (kavyashivakumar) [reply Requirements/Requirements_And_Goals.md:42]: that was for my sake. I didn't understand this requirement in the process. will add into questions to discuss

File: Requirements/Requirements_And_Goals.md (hunk 10 above)
Comment (kavyashivakumar) [reply Requirements/Requirements_And_Goals.md:49]: like gov, co.uk etc. (extracted from requirement)

File: Requirements/Requirements_And_Goals.md (hunk 11 above)
Comment (kavyashivakumar) [reply Requirements/Requirements_And_Goals.md:65]: what's this? didn't understand

File: Requirements/Requirements_And_Goals.md (hunk 2 above)
Comment (mgasca) [reply Requirements/Requirements_And_Goals.md]: I think either is fine.

File: Requirements/Requirements_And_Goals.md (hunk 4 above)
Comment (mgasca) [reply Requirements/Requirements_And_Goals.md:27]: Ok, still better to choose one and be consistent ourselves

File: Requirements/Requirements_And_Goals.md (hunk 7 above)
Comment (mgasca) [reply Requirements/Requirements_And_Goals.md:40]: 3rd Party Integration?

File: Requirements/Requirements_And_Goals.md (hunk 10 above)
Comment (mgasca) [reply Requirements/Requirements_And_Goals.md:49]: Can we clarify the txt here please?

File: Requirements/Requirements_And_Goals.md (hunk 11 above)
Comment (mgasca) [reply Requirements/Requirements_And_Goals.md:65]: Allow Police officers to connect is not a requiremnt of a charity is it?

Comment (kavyashivakumar) [reply Requirements/Requirements_And_Goals.md:65]: It is. according to the charity process in requirement doc

Comment (mgasca) [reply Requirements/Requirements_And_Goals.md:65]: seems strange but ok

**Pull request merged**